### DISCUSSION TITLE
Changing the sort order in the sidebar

### DISCUSSION THREAD

**Discussion by RS-JS** [chipzoller/hugo-clarity#227] [Q&A]
Hi, I want to change the sort order in the sidebar for both featured and most recent posts to show the most recent at the top by date. 
`     {{ range (where .Site.RegularPages "Section" "post").ByDate.Reverse }} `
I have added the code above to the recent post block and this works. I can't seem to get the same code working for the featured posts.

**Comment by chipzoller** [comment]
Featured is being overwritten by date. @onweru

**Comment by RS-JS** [comment]
I don't know if I need to stick this under feature requests, but it would be nice to have options to define the sort for different sections across the theme. The articles I am writing, I am using weight to split out what are chapters, appendixes, bibliographies etc, which is not necessarily the same as the date order they appear.  For my purposes, It would be nice when clicking on the archive tag they were sorted by weight, so they appeared in their intended order, and by date descending for the front page, and in the sidebar lists for featured, recent and related (thankyou for adding the latter btw)

**Reply by chipzoller** [reply]
It may be nice to have this as a feature request with a bit more clarification on the request. I'm still not quite clear as to how you're desiring this to look.

**Reply by RS-JS** [reply]
I will together something  in the new year, merry christmas
